integration/util/scenario/eth.js

```diff
@@ -5,13 +5,15 @@ const { readContractsFile, deployContract, getContractAt } = require('../ethereu
 const { genPort } = require('../util');
 
 class Eth {
-  constructor(ethInfo, web3, web3Url, accounts, ganacheServer, ctx) {
+  // TODO: Do the construct
+  constructor(ethInfo, web3, web3Url, accounts, ganacheServer, version, ctx) {
```

Comment:
Solidity contracts are now versioned so we can deploy older contracts for upgrade and compatibility tests.